test_scripts/API/Restructuring_OnResetTimeout/018_BC_OnResetTimeout_for_second_request_data.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC_1 is requested
+-- 2) RPC_1 is requested one more time
+-- 3) Some time after receiving RPC_1 requests on HMI is passed
+-- 4) HMI sends BC.OnResetTimeout(resetPeriod = 3000) to SDL for second request
+-- 5) HMI does not respond
+-- SDL does:
+-- Respond in 10 seconds with GENERIC_ERROR resultCode to mobile app to first request
+-- Respond in 13 seconds with GENERIC_ERROR resultCode to mobile app to second request
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function diagnosticMessageError()
+	local cid = common.getMobileSession():SendRPC("DiagnosticMessage",
```

Comment:
Please rename `local cid` and `local corId` to something more informative, like `local cidFirstReq` and `local cidSecondReq`